CMakeLists.txt

```diff
@@ -235,7 +242,12 @@ set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/Modules/")
 set(CMAKE_INSTALL_PREFIX ${CMAKE_CURRENT_BINARY_DIR})
 set(ARCHIVE_OUTPUT_DIRECTORY ./bin)
 
-set(CMAKE_CXX_FLAGS "-fPIC -std=gnu++0x -Wall -Werror -Wuninitialized -Wvla")
+set(CMAKE_CXX_FLAGS "-fPIC -std=gnu++11  -Wall -Werror -Wuninitialized -Wvla")
+
+get_os(OS)
+if(${OS} STREQUAL "qnx")
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -stdlib=libstdc++")
```

Comment:
@BSolonenko is this option applicable only to QNX? may be we can make it without if

Reply:
@AKalinich-Luxoft No, we can`t. This option is applicable only to QNX.